adapters/dynamoDB_adapters/pom.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>parent</artifactId>
+        <groupId>org.atomhopper</groupId>
+        <version>1.2.35-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+
+    <groupId>org.atomhopper.adapter</groupId>
+    <artifactId>dynamoDB_adapters</artifactId>
+    <packaging>jar</packaging>
+
```

Comment:
Add a `name` attribute matching the format of the other adapters. Example [link](https://github.com/rackerlabs/atom-hopper/blob/37f01063cec90c54b260e7caefa238f7a62d3a71/adapters/jdbc/pom.xml#L16).